terraform/root_common.tf

```diff
@@ -0,0 +1,307 @@
+module "global_parameters" {
+  source = "./tdr-configurations/terraform"
+}
+
+module "encryption_key" {
+  source      = "./tdr-terraform-modules/kms"
+  project     = var.project
+  function    = "encryption"
+  environment = local.environment
+  common_tags = local.common_tags
+  key_policy  = "message_system_access"
+}
+
+module "jenkins_ami" {
+  source      = "./tdr-terraform-modules/ami"
+  project     = var.project
+  function    = "ecs-ec2"
+  environment = local.environment
+  common_tags = local.common_tags
+  region      = var.default_aws_region
+  kms_key_id  = module.encryption_key.kms_key_arn
+  source_ami  = data.aws_ami.ecs_ami.id
+}
+
+module "jenkins_logs_s3" {
+  source        = "./tdr-terraform-modules/s3"
+  project       = "tdr"
+  function      = "jenkins-logs"
+  access_logs   = false
+  bucket_policy = "alb_logging_euwest2"
+  common_tags   = local.common_tags
+}
+
+module "jenkins_backup_s3" {
+  source      = "./tdr-terraform-modules/s3"
+  project     = "tdr"
+  function    = "jenkins-backup"
+  common_tags = local.common_tags
+}
+
+module "ecr_jenkins_repository" {
+  source           = "./tdr-terraform-modules/ecr"
+  name             = "jenkins"
+  image_source_url = "https://github.com/nationalarchives/tdr-jenkins/blob/master/docker/Dockerfile"
+  common_tags      = local.common_tags
+  policy_name      = "jenkins_policy"
+  policy_variables = { role_arn = module.jenkins_integration_execution_role.role.arn }
+}
+
+module "jenkins_integration_ecs_task_role" {
```

Comment:
What I was going to do was to have the production roles called `jenkins_prod_ecs_task_role` These are in the `root_common` but I think they should be in `root_integration` because the two Jenkins instances need different task roles. 
I've moved these to the root_integration file anyway.